hack/docs/internal/cronjob-tutorial/generate_cronjob.go

```diff
@@ -399,36 +400,36 @@ func (sp *Sample) updateWebhookTests() {
 	err = pluginutil.ReplaceInFile(file,
 		webhookTestsBeforeEachOriginal,
 		webhookTestsBeforeEachChanged)
-	hackutils.CheckError("replace validating defaulting test", err)
+	hackutils.CheckError("replace before each webhook test ", err)
 }
 
 func (sp *Sample) updateWebhook() {
 	var err error
 	err = pluginutil.InsertCode(
-		filepath.Join(sp.ctx.Dir, "api/v1/cronjob_webhook.go"),
+		filepath.Join(sp.ctx.Dir, "internal/webhook/v1/cronjob_webhook.go"),
 		`limitations under the License.
 */`,
 		`
 // +kubebuilder:docs-gen:collapse=Apache License`)
 	hackutils.CheckError("fixing cronjob_webhook.go by adding collapse", err)
 
-	err = pluginutil.ReplaceInFile(
-		filepath.Join(sp.ctx.Dir, "api/v1/cronjob_webhook.go"),
+	err = pluginutil.InsertCode(
+		filepath.Join(sp.ctx.Dir, "internal/webhook/v1/cronjob_webhook.go"),
 		`import (
 	"context"
-	"fmt"`, `import (
-	"context"
-	"fmt"
+	"fmt"`,
+		`
```

Comment:
Nit: I think its taken care by running `go fmt` in the end, but the file still has `fmt` and `context` as imports. So probably we can revert back the first two statements? (https://github.com/kubernetes-sigs/kubebuilder/blob/d6f6c346bb1010880af1e70fabf31623fb816a82/docs/book/src/cronjob-tutorial/testdata/project/internal/webhook/v1/cronjob_webhook.go#L21-L22)

Reply:
Good catcher 🥇

Reply:
Previously, we were using Replace. Now, we are using Insert to be more precise and avoid breaking the code. This allows us to insert the rest of the text starting from a specific point. So, I think that is right, but let me know if I  missing something.